app/Post_Types/Archive_Item/Post_Meta.php

```diff
@@ -146,6 +146,19 @@ public function add_dynamic_fields(){
 
 			}
 		}
+
+		// add the related field
+		$related = carbon_get_theme_option(\Diviner\Admin\Settings::FIELD_GENERAL_RELATED_FIELD);
+		if ( $related ) {
+			$field = \Diviner\Post_Types\Diviner_Field\Types\Related_Field::render(
+				0,
+				'diviner_related',
+				'Related Archive Items',
+				'Appears on each archive item single page as a slider'
+			);
+			$dyn_fields[] = $field;
```

Comment:
let's be explicit here: `dynamic_fields` in place of `dyn_fields`, it's not that longer and more readable.

Reply:
Done